### PR TITLE
Closes #113 - replaced node-fetch with axios

### LINKED ISSUE #113: Update project to use axios for making http requests
Node-fetch has switched to using esm and explicit paths in imports which are breaking changes https://github.com/node-fetch/node-fetch/issues/1397. So it was decided to axios for now. Maybe in the future we will use node-fetch again

### PR DESCRIPTION
**Pull request by IngridGdesigns**
removed node-fetch and type
added axios and edited code
added DOM in compilerOptions to deal with typescript error

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
         "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
         "lib": [
             "es2019",
-            "ES6"
+            "ES6",
+            "DOM"
```
Comment (kevindigo) [inline tsconfig.json:9]: Do we really need to add DOM? That surprises me since we're not doing anything browser-ish.

**Review by kevindigo — CHANGES_REQUESTED** [review]
Wow. That was almost a direct drop-in replacement. Sweet!

File: tsconfig.json (hunk 1 above)
Comment (IngridGdesigns) [reply tsconfig.json:9]: on my end yes, I was getting errors if it wasnt present, not sure if there is another solution, will do some research

Comment (IngridGdesigns) [reply tsconfig.json:9]: so the error I was getting before adding DOM in was :
```
 TS2304: Cannot find name 'AbortSignal'.

108   signal?: AbortSignal;
               ~~~~~~~~~~~
```

Comment (IngridGdesigns) [reply tsconfig.json:9]: @kevindigo updated dependencies in lock file, and the error ceased to pop up when i run  `npm run build`

Comment (IngridGdesigns) [reply tsconfig.json:9]: to be clear, DOM was removed

**Review by keeganwitt — APPROVED** [review]

**Pull request merged**